pandas/io/html.py

```diff
@@ -606,18 +604,20 @@ def _build_doc(self):
         """
         from lxml.html import parse, fromstring, HTMLParser
         from lxml.etree import XMLSyntaxError
-
-        parser = HTMLParser(recover=False, encoding=self.encoding)
+        parser = HTMLParser(recover=True, encoding=self.encoding)
```

Comment:
This is a change in behavior giving lxml a little more power to work through malformed HTML. May or may not be acceptable (see other comments)